candle-nn/src/var_builder.rs

```diff
@@ -497,7 +497,7 @@ impl<'a> VarBuilder<'a> {
     ///
     /// # Safety
     ///
-    /// The unsafe is inherited from [`memmap2::MmapOptions`].
+    /// The unsafe is inherited from [`candle::safetensors::MmapedSafetensors::multi`].
```

Comment:
```
   --> /home/ivor/Documents/Code/rust/playground_candle/candle/candle-nn/src/var_builder.rs:500:40
    |
500 |     /// The unsafe is inherited from [`memmap2::MmapOptions`].
    |                                        ^^^^^^^^^^^^^^^^^^^^ no item named `memmap2` in scope
    |
    = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default
```

Presumably a copy paste from the documentation of [multi](https://github.com/huggingface/candle/blob/242e006bbb26ff12581b3c04bfd069996fe1f6bb/candle-core/src/safetensors.rs#L288-L294) itself, changed to point at the reason why this function is unsafe.